newFile.py

```diff
@@ -0,0 +1,22 @@
+stages:
```

Comment:
Reviewing changes in **newFile.py**:

This code establishes a pipeline stage called `pr_agent` and sets up the job for a PR Agent action with multiple environment variables and few action steps, it needs a newline character at the end.